src/styles/blocks/_button.scss

```diff
@@ -0,0 +1,32 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  width: max-content;
+
+  padding: 15px 74px;
+
+  @include font-param(16px, 27px, 700);
+  font-family: inherit;
+  color: #fff;
+
+  border: 1px solid transparent;
+  background-color: $main-accent;
+
+  @include for-mobile {
+    width: 100%;
```

Comment:
>11	Make sure everything looks neat on mobile and 
> without horizontal scrolling

![mobile horizontal scroll](https://user-images.githubusercontent.com/39308717/163112111-af029e20-ce35-4d8e-a4d4-e05a74f217e4.png)

Reply:
@meforoma It works when I set mobile from list